### ISSUE TITLE
Wrong font on submit button

### ISSUE THREAD

**Issue by csell5** [ThatConference/that-website#130]
On https://www.thatconference.com/wi/become-a-partner the form submit button is a different font weight than the rest of the site's buttons.

![image](https://user-images.githubusercontent.com/772569/70849110-c8459e00-1e3f-11ea-85a2-2f393cb75314.png)

compared to

![image](https://user-images.githubusercontent.com/772569/70849122-e7443000-1e3f-11ea-9546-07a7e8400a9f.png)

**Comment by BradCandell** [comment]
@brettski - I talked to @csell5 about this one, and it looks like it will require changes to the style within AC. 

Using the information from `LinkButton.js` and the styles from the generated AC form, the following style modifications *should* make the Submit button look like the rest of the style:

```css
    font-size: 0.9375rem;
    min-width: 20rem;
    margin-top: 2rem;
    width: auto;
    height: auto;
    padding: 1.3rem !important;
    font-weight: 400;
    border: 2px solid #fff !important;
```

Since the `<LinkButton />` uses an inner `<p>` tag with margin and AC uses a Submit button itself, I just used padding instead.

Let me know if you have any questions.
